### PR TITLE
Fix linking on Rust versions that unprefix jemalloc

### LINKED ISSUE #43: linking issue on trusty with and latest nightly
linking of application using heapsize and built with latest nightly fails on ubuntu trusty

example:

https://github.com/debris/heapsize_issue

travis build matrix:

https://travis-ci.org/debris/heapsize_issue

### PR DESCRIPTION
**Pull request by SimonSapin**
Fixes #43, fixes #44.

[Review on Reviewable](https://reviewable.io/reviews/servo/heapsize/46)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
+
 notifications:
   webhooks: http://build.servo.org:54856/travis
 
 script:
  - cargo test
- - "[ $TRAVIS_RUST_VERSION = nightly ] || cargo test --features unstable"
- - "[ $TRAVIS_RUST_VERSION = nightly ] || cargo test --manifest-path plugin/Cargo.toml"
+ - "[ $TRAVIS_RUST_VERSION != nightly ] || cargo test --features unstable"
+ - "[ $TRAVIS_RUST_VERSION != nightly ] || cargo test --manifest-path plugin/Cargo.toml"
```
Comment (nox) [inline .travis.yml]: This won't test unstable and plugin on `nightly-*` though, right?

Comment (SimonSapin) [inline .travis.yml]: Right, but the point of having these two dated version is to test the build script’s version parsing and the linking to `je_malloc_usable_size`. These versions are not special as far as the `unstable` feature and the plugin are concerned.

**Pull request merged**